main/perl-mail-clamav/APKBUILD

```diff
@@ -14,13 +14,19 @@ makedepends="bzip2-dev clamav-dev zlib-dev
 	"
 install=
 subpackages="$pkgname-doc"
-source="http://search.cpan.org/CPAN/authors/id/C/CO/CONVERTER/$_realname-$pkgver.tar.gz"
+source="http://search.cpan.org/CPAN/authors/id/C/CO/CONVERTER/$_realname-$pkgver.tar.gz
+	0001.patch"
 
 _builddir="$srcdir/$_realname-$pkgver"
 
 prepare() {
 	cd "$_builddir"
 	sed -i -e 's%\(clamav-config --version\)%sh /usr/bin/\1%' Makefile.PL || return 1
+	for i in $source; do
```

Comment:
You can call "default_prepare" at the begin of prepare() function so it will apply your patch. Don't need this for.